packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -28,7 +28,9 @@ export const richTextEditorWrapperStyle = (theme: Theme): string => {
     paddingInlineEnd: `0.75rem`,
     maxWidth: '100%',
     color: theme.palette.neutralPrimary,
-
+    '& img': {
+      margin: '0.2rem'
+    },
```

Comment:
Could we also apply this to the edit box as well?

Reply:
Won't this apply to edit box too? As both sendbox and editbox use this?